tests/test_rhymes.py

```diff
@@ -470,3 +490,605 @@ def test_analyze_rhyme_romance(romance):
+    ningún remedio se alcanza,
+    pues me matan la esperanza
+    desdenes, celos y ausencia."""
+    output = "ovillejo"
+    input_poem = get_scansion(poem, rhyme_analysis=True)
+    assert input_poem[0]["structure"] == output
+
+
+def test_aleluya():
+    poem = """¡Soria fría, Soria pura,
+    cabeza de Extremadura,
+    con su castillo guerrero
+    arruinado, sobre el Duero;
+    con sus murallas roídas
+    y sus casas denegridas!"""
+    output = "aleluya"
+    input_poem = get_scansion(poem, rhyme_analysis=True)
+    assert input_poem[0]["structure"] == output
+
+
+def test_quinteto():
+    poem = """Juegan y beben: mas en bien, sin vicio,
+    sin interés y sin exceso: tienen
+    del cuarto de Fermín mal en el quicio
+    encajada la puerta y se mantienen
+    ojo avizor a él por el resquicio."""
+    output = "quinteto"
+    input_poem = get_scansion(poem, rhyme_analysis=True)
+    assert input_poem[0]["structure"] == output
+
+
+def test_sexteto():
+    poem = """Entre las rocas de la costa alzada
+    se oye un extraño hablar, de madrugada,
+    de gentes que en la noche vigilaron;
+    las barcas, animadas de un deseo,
+    tienen un misterioso balanceo,
+    y nunca se están quietas en donde las dejaron."""
+    output = "sexteto"
+    input_poem = get_scansion(poem, rhyme_analysis=True)
+    assert input_poem[0]["structure"] == output
+
+
+def test_sextilla():
+    poem = """Existe una poesía
+    sin ritmo ni armonía
+    monótona, cansada,
+    como una letanía...,
+    de que está desterrada
+    la pena y la alegría."""
+    output = "sextilla"
+    input_poem = get_scansion(poem, rhyme_analysis=True)
+    assert input_poem[0]["structure"] == output
+
+
+def test_septeto():
+    poem = """Vengo a mirarte, campo doloroso,
+    cuando son triste leña tus encinas,
+    cuando en rigores de tu polvo inclinas
+    sus mutilados miembros al reposo
+    y en las huellas del ciervo sin camino
+    se alberga el yerto ruiseñor piadoso
+    segado, en pluma tierna, de su trino."""
+    output = "septeto"
+    input_poem = get_scansion(poem, rhyme_analysis=True)
+    assert input_poem[0]["structure"] == output
+
+
+def test_septilla():
+    poem = """Luz de sueño, flor de mito,
+    tu admirable cuerpo canta
+    la gracia de Hermafrodito
+    con lo aéreo de Atalanta;
+    y de tu beldad ambigua
+    la evocada musa antigua
+    su himno de carne levanta."""
+    output = "septilla"
+    input_poem = get_scansion(poem, rhyme_analysis=True)
+    assert input_poem[0]["structure"] == output
+
+
+# def test_octavilla():
+#     poem = """La mayor cuita que haber
+#     puede ningún amador
+#     es membrarse del placer
+#     en el tiempo del dolor;
+#     e ya sea que el ardor
+#     del fuego nos atormenta,
+#     mayor dolor nos aumenta
+#     esta tristeza y langor."""
+#     output = "octavilla"
+#     input_poem = get_scansion(poem, rhyme_analysis=True)
+#     assert input_poem[0]["structure"] == output
```

Comment:
Remove?

Reply:
Yeah it's one of the stanzas I was having problems with, will remove and add later when it works